### DISCUSSION TITLE
blog/cloud-gpu-services-jupyter-notebook-reviewed

### DISCUSSION THREAD

**Discussion by giscus** [zackproser/portfolio#848] [Announcements]
# blog/cloud-gpu-services-jupyter-notebook-reviewed

I tried a handful of services when I last needed to fine-tune an LLM, and I was mostly disappointed...

https://zackproser.com/blog/cloud-gpu-services-jupyter-notebook-reviewed

**Comment by jxrgameshub** [comment]
Haha I can so relate to this bro. Just this morning I wanted to go back to sleep after trying to use gradient gpu for a lightweight finetune that supposedly only takes an hour.
